etl/gazetteer_sql.mk

```diff
@@ -0,0 +1,42 @@
+# Load Gazetteer data into a SQL database.
+
+# The table name used in the database begins with "gaz_" instead of matching the input filename
+# ("2019_"), so that the table name begins with a letter, so that the table name
+# does not need to be quoted when used in a SQL query.
+
+GAZETTEER_AREA_TYPES = counties_national cosubs_national place_national
+GAZETTEER_YEARS = 2019 2020 2021
+
+define GAZETTEER_LOAD_DATA_RULE
+# GAZETTEER_LOAD_DATA_RULE - Load raw gazetteer dataset into database.
+# Parameters:
+#   - $1 = Year
+#   - $2 = Area Type
+$(TABLE_PROXY_DIR)/gaz_$1_$2: $(DATA_DIR_SRC)/$1_Gaz_$2.txt | $(TABLE_PROXY_DIR)
+	@# Gazetteer files have trailing whitespace at the end of each line
+	@# which results in the colum name 'INTPTLONG                  '.
+	pipenv run bash -c ' \
+	    ./etl/trim_trailing_tsv_whitespace.py $$< | \
+	    sqlite-utils insert --tsv \
+	        --pk GEOID $(SQLITE_DB_PATH) $$(basename $$(notdir $$@)) -' \
```

Comment:
I'd like to suggest adding `--truncate` to the `sqlite-utils insert` calls so that the table will be emptied before inserting. This would allow someone to re-populate the table just by deleting/remaking the table proxy file. Currently, without the `--truncate`, these SQL targets error out (correctly) with an error about duplicate primary keys.

Reply:
This makes sense to me if you want to add these.

Reply:
I added this to all instances of `sqlite-utils insert` in c4601407d6e9cc33fd08ac90329d96ad4f86d642. It looked like all of these were intended to load complete datasets into a table, as opposed to incrementally loading data into a table. (But let me know if this isn't the case).